.travis.yml

```diff
@@ -3,11 +3,12 @@ language: ruby
 rvm:
   - jruby-9.1.9.0
   - jruby-9.2.9.0
-  - 2.4
   - 2.5
   - 2.6
   - 2.7
+  - 3.0
 env:
   JRUBY_OPTS=--2.0
+  RUBYOPT="-W0"
```

Comment:
This is setting verbose mode to "silent". Is that what we want to do here?

Reply:
I was trying to silence the many, many warnings I've been getting about URI, but this didn't work. Do you have advice about a better solution?